ui/docs/manual/using/integration/frontend.md

```diff
@@ -9,52 +9,32 @@ order: 10
 A frontend application -- one which executes in the browser -- can easily make
 calls to Taskcluster APIs using the user's credentials.
 
-The application should handle user logins using the normal Mozilla process --
-currently using OpenID Connect via Auth0 and following the IAM team's
-[recommendations](https://wiki.mozilla.org/Security/Guidelines/OpenID_connect).
-This can support either a single page application
-(["SPA"](https://auth0.com/docs/clients#client-types)) with no backend or a
-hybrid ("[regular web
-application](https://auth0.com/docs/clients#client-types)").
-
-This process supports any authentication and authorization your application
-needs for itself - displaying the user's name, storing user settings, or
-controlling access to resources based on the user identity.
-
-With a few extra parameters to the login process, this process will produce an
-access token which can also be exchanged for Taskcluster credentials as they
-are needed, by making an API call to the [login
-service](/docs/reference/integrations/taskcluster-login).
-
-The Taskcluster credentials have a very short expiration, but can be requested
+Taskcluster implements the [OAuth2 protocol](https://tools.ietf.org/html/rfc6749),
+supporting both the "Implicit" and "Authorization Code" flows.
+The "Resource Owner Password Credentials" and "Client Credentials" flows are not supported.
+
+Clients are pre-defined, and each pre-defined client indicates which flow it uses (and cannot use both).
+Some clients are whitelisted, meaning that user consent is not required.
```

Comment:
Let's link to [`/docs/manual/deploying/third-party`](https://github.com/taskcluster/taskcluster/pull/1202/files#diff-9562a34c774f1c8c5ea1e0c30f5d64a5) here so that readers can figure out how registration occurs.